README.md

```diff
@@ -19,12 +19,12 @@
    ```
    ````
 1. Write the post and select appropriate categories. Tags are not available.
-1. Add required packages to DESCRIPTION file via `usethis::use_package(<package>)`.
-1. Preview locally using `rmarkdown::render_site()` and `pkgdown::preview_site()` or (when using RStudio) by clicking on "Build Website" in the "Build" pane.
+1. Add required packages to the DESCRIPTION file via `usethis::use_package(<package>)`.
+1. Preview your post locally using `rmarkdown::render()`.
```

Comment:
Hm no, this was not what I meant.

1. Render the Rmd to create the HTML
1. Preview via `rmarkdown::render_site()` and `pkgdown::preview_site()`

Reply:
I guess I missunderstood you here. I reverted these parts of the README.